### DISCUSSION TITLE
Using Closed Source Solutions as a part of plugins

### DISCUSSION THREAD

**Discussion by rubenvdlinde** [OpenWebconcept/Afspraken#15] [General]
I recently discovered that OpenPub relies on https://metabox.io/plugins/meta-box-group/ wich has an pricing and licensing model that in my opinion is incompatible with Open Webconcept way of working. This prevents me from using PUB in an non-commercial sandbox environment without actually installing and paying for commercial solutions.

The reason why this plugin was added as a dependency also eludes me, it only seems to be used for grouping forms in the admin frontend. Something that could have fairly easily been coded (to my opinion)

I would like to make a project wide design decision that plugins can not have a HARD dependency on commercial close source solutions that require third party licensing.

**Comment by Htbaa** [comment]
The same seems to be the case for OpenPDC, which requires MB Settings Page in addition to Meta Box Group to be properly setup, yet it doesn't mention this in its requirements.

I fully agree with you.

**Comment by Htbaa** [comment]
It also seems OpenPub requires ElasticPress, but is ElasticSearch really required for OpenPub to function?

**Comment by mvdhoek1** [comment]
Hi everyone,

Since version 3.0.0 Metabox.io is replaced with the CMB2 plugin. The dependency on ElasticPress has been removed as well. 
CMB2 will be implemented in to other plug-ins in the nearby future.
